test/SerialIOTest.cpp

```diff
@@ -1171,7 +1176,9 @@ TEST_CASE( "adios1_dtype_test", "[serial][adios1]" )
     REQUIRE(s.getAttribute("uint64").get< uint64_t >() == 64u);
     REQUIRE(s.getAttribute("float").get< float >() == 16.e10f);
     REQUIRE(s.getAttribute("double").get< double >() == 1.e64);
+#if !defined(_MSC_VER)
     REQUIRE(s.getAttribute("longdouble").get< long double >() == 1.e80L);
```

Comment:
Why not use ```std::numeric_limits<long double>::max()``` here as well? Is it the initializer list causing the problems?

Reply:
The problem I caused by the `long double` access in #179 
I will exclude it for now and open a PR that removes the exclusion to demonstrate.